### PR TITLE
[Update] update doc about seeding

### PR DESCRIPTION
**Pull request by kenzo0107**
@lorenzo I added a note to docs/seeding.rst. Please review.

Add a notice about bulk-insert operation to docs/seeding.rst.

related [#feature/bulkinsert]

### CODE REVIEW

**Comment by JayPHP** [comment]
This addition doesn't really make sense, grammatically speaking and needs to be a bit clearer - Could you open an issue for this please?

**Comment by kenzo0107** [comment]
Thank you for your review.
I open an issue for this.

https://github.com/cakephp/phinx/issues/1194

**Comment by dereuromark** [comment]
Why closing? Just needed a small tip on how to fix the grammar issue, then this could have been merged.

**Comment by JayPHP** [comment]
@dereuromark because it wasn't just a small tip, the sentence didn't make any sense and there was no issue for reference.

**Comment by dereuromark** [comment]
True, the reference has now been added, maybe we can resolve this then together with @kenzo0107

File: docs/seeding.rst [hunk 1]
```diff
@@ -110,6 +110,8 @@ within your seed class and then use the `insert()` method to insert data:
 
     You must call the `save()` method to commit your data to the table. Phinx
     will buffer data until you do so.
+    You call the `save()` method and if all the keys of each data are the same,
```
Comment (dereuromark) [inline docs/seeding.rst]: If you call ...

File: docs/seeding.rst [hunk 2]
```diff
@@ -110,6 +110,8 @@ within your seed class and then use the `insert()` method to insert data:
...
+    Phinx execute bulk-insert operation automatically to improve the performance.
```
Comment (dereuromark) [inline docs/seeding.rst]: executes a bulk-insert

**Comment by dereuromark** [comment]
ping @kenzo0107

**Comment by kenzo0107** [comment]
@dereuromark I'm so sorry. I didn't notice your comment.
Thank you for the review.

I update seeding.rst a little.

File: docs/seeding.rst [hunk 3]
```diff
@@ -110,6 +110,8 @@ within your seed class and then use the `insert()` method to insert data:
 
     You must call the `save()` method to commit your data to the table. Phinx
     will buffer data until you do so.
+    If you call the `save()` method and all the keys of each data are the same,
+    Phinx executes bulk-insert and improve the performance.
```
Comment (dereuromark) [inline docs/seeding.rst]: improves

**Comment by kenzo0107** [comment]
@dereuromark I updated it.

**Comment by dereuromark** [comment]
Might need a small rebase.

**Comment by othercorey** [comment]
Closing due to no activity.

**Pull request closed without merging**